.travis.yml

```diff
@@ -40,6 +41,7 @@ branches:
   only:
   - master
   - /^dawn\/.*$/
+  - /^hibike\/.*$/
```

Comment:
This list is only for pushing releases. hibike currently doesn't have anything that will be released, so no need for this I think.